pages/common/pdfimages.md

```diff
@@ -0,0 +1,20 @@
+# pdfimages
+
+> Utility for extracting images from PDFs.
+
+- Extract all images from a PDF file and save them in PNG format:
+
+`pdfimages -png {{PDF-file}} {{image-root}}`
+
+- Extract images from a PDF file from pages 3-5:
+
+`pdfimages -f 3 -l 5 {{PDF-file}} {{image-root}}`
+
+- Extract images from a PDF file and include the page number in their names:
+
+`pdfimages -p {{PDF-file}} {{image-root}}`
```

Comment:
Perhaps this will read better?
```suggestion
`pdfimages -p {{path/to/file.pdf}} {{path/to/output_directory}}`
```

Same goes for the other examples.

Reply:
Thanks for all the suggestions! I've committed all of them, except this one. The last parameter here is not a path to the output directory, but rather a name root, that all of the resulting filenames will share. I.e., running the command
```
pdfimages -png file.pdf test
```
will produce files
```
test-001.png
test-002.png
...
```
rather than saving them in a directory named `test`.

I agree it is not readily understandable from the token `image-root`, but I can't think of anything better and that's the term the program and its man page uses.

Reply:
Ah, I see! Thanks for clarifying. What about `filename_prefix` then?

Reply:
Good idea; implemented.